perf/benchmark/templates/fortio.yaml

```diff
@@ -206,13 +209,15 @@ spec:
           number: 8080
 ---
 {{- end }}
+{{- end }}
 
 {{- $fortioserver := dict "name" "fortioserver" "Values" .Values "V" .Values.server}}
 {{- template "fortio" $fortioserver }}
 {{- $fortioclient := dict "name" "fortioclient" "Values" .Values "V" .Values.client }}
 {{- template "fortio" $fortioclient }}
 
 ---
+{{- if $.Values.server.inject }}
```

Comment:
this should be Values.server.inject or Values.client.inject?

Reply:
either is ok